src/cli-access-protected-api/cli.py

```diff
@@ -70,6 +71,7 @@ def main(tenant: str, client_id: str, object_id: str, secret: str):
     )
 
     # First check for an existing token in the cache and/or refresh if needed
+    # <ms_docref_get_graph_token>
```

Comment:
@henrymbuguakiarie Please include descriptive comments like this in the named snippets.

Cc: @ckittel @Dickson-Mwendia